Automattic/jetpack
# JS Components: Add adaptive coloring to the donut meter

#### Changes proposed in this Pull Request:

Adds togglable adaptive coloring to the donut meter that follows these rules:
- Percentage < 50% will result in a green meter.
- Percentage > 50% but < 100% will result in a yellow meter.
- Percentage > 100% will result in a red meter.

<img width="346" alt="image" src="https://user-images.githubusercontent.com/4044428/194440869-abeebc4c-a22c-460d-99ea-8619b3c02b31.png">


#### Other information:

#### Jetpack product discussion


See pcNPJE-140-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Spin up your storybook instance (`cd projects/js-packages/storybook && pnpm run storybook:dev`
* Navigate to the donut meter section
* Ensure each example renders as expected.
